release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -192,6 +192,21 @@ For more information, see xref:../support/gathering-cluster-data.adoc#support-ne
 
 [id="ocp-4-10-hardware"]
 === Hardware
+=======
```

Comment:
Looks like this ======= bit is here by mistake, possibly leftover from a resolved merge conflict.